.goreleaser.yml

```diff
@@ -4,23 +4,27 @@ builds:
 - binary: summon2
   env:
   - CGO_ENABLED=0
+  flags:
+  - -tags=netgo
   goos:
   - linux
   goarch:
   - amd64
   id: summon2
+  ldflags: -s -w -linkmode external -extldflags "-static"
```

Comment:
1. summon2 was not statically built so it could only work on musl-lib systems (Alpine mostly). This change adds static building so that it can run anywhere. Additionally, we pass the git short-hash through the ldflags to the broker now.

2. The only thing different now between summon2 and the broker in terms of flags is the ldflags that sets the short-hash for the broker. Summon2 has no support for this feature (yet).

Reply:
Resolving this since it carried no direct need for changes